unittest/packaging/cmake/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 cmake_minimum_required(VERSION 2.6)
 
 project(ExtraLib CXX)
-find_package(pinocchio REQUIRED)
 
 set(PINOCCHIO_MODEL_DIR "${PROJECT_SOURCE_DIR}/../../../models")
 add_definitions('-DPINOCCHIO_MODEL_DIR=\"${PINOCCHIO_MODEL_DIR}\"')
+set(EXAMPLE_ROBOT_DATA_MODEL_DIR "\"${PINOCCHIO_MODEL_DIR}/example-robot-data/robots\"")
+add_definitions('-DEXAMPLE_ROBOT_DATA_MODEL_DIR=\"${EXAMPLE_ROBOT_DATA_MODEL_DIR}\"')
```

Comment:
Why doing that if you don't update .cpp file ? (they still use PINOCCHIO_MODEL_DIR)

Reply:
fixed in 4793f45ed5aeffcd875b88c8940979c431ad46bc